experimental/segmenter/src/provider.rs

```diff
@@ -97,23 +106,25 @@ pub struct UCharDictionaryBreakDataV1<'data> {
 #[cfg_attr(feature = "serde", derive(serde::Deserialize))]
 #[yoke(prove_covariance_manually)]
 pub struct LstmMatrix<'data> {
+    #[allow(missing_docs)]
     #[cfg_attr(feature = "serde", serde(borrow))]
     pub dim: ZeroVec<'data, i16>,
+    #[allow(missing_docs)]
     #[cfg_attr(feature = "serde", serde(borrow))]
     pub data: ZeroVec<'data, f32>,
```

Comment:
Question: I don't know lstm well enough to document these two fields. @sffc or @makotokato Could you help on this?